composables/autoTeleport/useAutoTeleportTransitionDetails.ts

```diff
@@ -285,9 +286,8 @@ export default function (
             const address = getAddressByChain(currentChain.value as Chain)
             return getActionTransactionFee({
               api,
-              action: action,
+              action: action as Actions,
               address,
-              prefix: prefix as Prefix,
             })
```

Comment:
u sure with this?

Reply:
oh sorry, i readd back again the prefix args. same with above also